### DISCUSSION TITLE
my vim.cmd() is not working in config.lua but the command works when run manually?

### DISCUSSION THREAD

**Discussion by dcopenhaver** [LunarVim/LunarVim#4420] [General]
Hello,

I'm new to lunarvim (and neovim), I'm trying to get this working from my config.lua file:
vim.cmd("highlight CursorLine gui=underline cterm=underline")

If already running lvim, I can run this from command mode and it works as expected:
:highlight CursorLine gui=underline cterm=underline

However with vim.cmd("highlight CursorLine gui=underline cterm=underline") in my config.lua nothing happens?
~/.config/lvim/config.lua

Pretty much out of box config of lvim right now other than one plugin for the darkplus colorscheme. It's short so here's my config.lua file now:

```-- my plugins
lvim.plugins = {
  { "lunarvim/darkplus.nvim" }
}

-- my general options
lvim.colorscheme = "darkplus"
lvim.builtin.lualine.options.theme = "darkplus"

vim.cmd("highlight clear CursorLine")
vim.cmd("highlight CursorLine gui=underline cterm=underline")
```

**Comment by LostInTheLogs** [comment]
vim.cmd does work, but the highlights are overwritten when the colorscheme is loaded, you need to run the vim.cmd after the colorscheme is changed like it is done [in the docs](https://www.lunarvim.org/docs/configuration/appearance/colorschemes#customizing-some-colors)

**Reply by dcopenhaver** [reply]
Thanks LostNeophyte, that did the trick. I saw that in the docs before but I guess I'm to new to lvim/neovim to even realize what it was telling me, hehe.

To anyone else that my stumble the same way, I added the following and it's doing exactly what I wanted now:

```
lvim.autocommands = {
  {
    { "ColorScheme" },
    {
      pattern = "*",
      callback = function()
        -- see ':h nvim_set_hl' for more options
        vim.api.nvim_set_hl(0, "CursorLine", { bg = "#282828", underline = true, bold = false })
      end,
    },
  },
}
```

**Reply by huynhj93** [reply]
Thanks for this, ran into the same issue and was having major issues. just a note for others: if you run into an error with bad parameters, it'll never reload as your init.lua crashed. i had to exit lvim and reload it to a state with no issues, then keep retrying with test parameters and repeat the step above if errored until one succeeds.

**Comment by dcopenhaver** [comment]
Implemented per LostNeophyte's suggestion (and the docs). Details in my previous comment.
